app/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,10 @@
   <string name="play_pause_audio">Play/Pause Audio</string>
   <string name="help">Help</string>
   <string name="preferences">Preferences</string>
+  <string name="fraction_error_invalid_chars">Please only use numerical digits, spaces or forward slashes (/)</string>
+  <string name="fraction_error_invalid_format">Please enter a valid fraction (e.g., 5/3 or 1 2/3)</string>
+  <string name="fraction_error_divide_by_zero">Please do not put 0 in the denominator</string>
+  <string name="fraction_error_larger_than_seven_digits">None of the numbers of the fraction should be larger than 7 digits.</string>
```

Comment:
Did you add any new tests for this case? Please do if not.

Reply:
Added.